src/core/net/nd6.hpp

```diff
@@ -84,6 +84,7 @@ class Option
         kTypeRouteInfo          = 24, ///< Route Information Option.
         kTypeRecursiveDnsServer = 25, ///< Recursive DNS Server (RDNSS) Option.
         kTypeRaFlagsExtension   = 26, ///< RA Flags Extension Option.
+        kTypeNat64Prefix        = 38, ///< NAT64 Prefix Option.
```

Comment:
```suggestion
        kTypeNat64Prefix        = 38, ///< NAT64 Prefix Option (aka PREF64 Option).
```

Here would be a good place to state the alternate name of the option. It has two names in RFC 8781; some people might only search for the name "PREF64" so these people will find this item. Since this name may be more commonly known due to it being in the title of the RFC and in the IANA registry.